lib/luadata.c

```diff
@@ -84,7 +94,7 @@ static int luadata_setstring(lua_State *L)
 	size_t length;
 	const char *str = luaL_checklstring(L, 3, &length);
 	luadata_t *data = luadata_checkdata(L, &offset, (lua_Integer)length);
-
```

Comment:
let's not remove these `\n`'s.. I've asked to remove them only from templates.. to "force" us to keep such macros small and coarse ;-).. avoiding doing too much on them and structure our code a bit better..